core/src/test/java/hudson/BulkChangeTest.java

```diff
@@ -46,17 +45,17 @@ private static class Point implements Saveable {
         @SuppressWarnings("unused")
         int x, y;
 
-        public void setX(int x) throws IOException {
+        public void setX(int x) {
             this.x = x;
             save();
         }
 
-        public void setY(int y) throws IOException {
+        public void setY(int y) {
```

Comment:
This seems unrelated. Given the size of the PR (even if split), I'd prefer to have no unrelated cleanups thrown in at the same time. Perhaps something you could consider for the resubmission.